kubernetes-steps/src/main/java/io/fabric8/kubernetes/pipeline/KubernetesFacade.java

```diff
@@ -141,7 +141,8 @@ public Pod createPod(String hostname, String jobname, PodTemplate podTemplate, S
                     .withWorkingDir(buildWorkspace)
                     .withCommand(split(c.getCommand()))
                     .withArgs(split(c.getArgs()))
-                    .withTty(c.isTtyEnabled())
+                    //.withTty(c.isTtyEnabled())
+                    .withTty(true)
```

Comment:
Yes, you are right. I did that because for me, I had the issue that I got an exception that the method "isTtyEnabled" doesn't exist. But to be correct, I rolled that back.